translate/cloud-client/translate_v3_translate_text_with_glossary.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_translate_text_with_glossary]
+
+from google.cloud import translate_v3
+
+
+def translate_text_with_glossary(
+    text, source_language, target_language, project_id, glossary_id
+):
+    """Translates a given text using a glossary."""
+
+    client = translate_v3.TranslationServiceClient()
+
+    # text = 'Hello, world!'
+    # source_language = 'en'
```

Comment:
nit: In Java we have a requirement where we only include arguments that are required for testing as arguments. Configurables like `source_language` or `target_langauge` would just be set to some default value, with a link to the API docs where supported values for that would be found (so users can see what other values can be used). 

This will (likely) be introduced into the Python sample requirements sometime soon as well.